generators/go/modules/dockerfile/Dockerfile.template

```diff
@@ -0,0 +1,29 @@
+FROM golang:{{{ .GoVersion }}}-alpine AS base
+
+RUN go install github.com/go-delve/delve/cmd/dlv@latest
+
+WORKDIR /build
+COPY go.mod .
+COPY go.sum .
+RUN go mod download
+
+COPY cmd cmd
+COPY internal internal
+
+FROM base AS builder
+
+WORKDIR cmd/{{{ .ProjectName }}}
```

Comment:
Can you change these lines to `{{{ _toDir .ProjectName }}}` here like we have in the generator config file? There are a couple premade helper funcs like `_toDir` to do things like sanitize inputs and give us something suitable for a directory name.